scalalib/overrides-2.11/scala/runtime/ScalaRunTime.scala

```diff
@@ -201,7 +200,7 @@ object ScalaRunTime {
     val high = (lv >>> 32).toInt
     low ^ (high + lowSign)
   }
-  def hash(x: Number): Int  = runtime.BoxesRunTime.hashFromNumber(x)
+  def hash(x: Number): Int = x.hashCode
```

Comment:
This is needed because of the removed `HashMethods` value mentioned below. I think that, as a result, the `hashFromNumber` method is not intrinsified anymore and the one in `BoxesRunTime` is not implemented.

Reply:
I don't know why they were removed in the first place. Maybe we could only remove them for 2.12.x onwards.

Reply:
> I don't know why they were removed in the first place. Maybe we could only remove them for 2.12.x onwards.

That would probably make more sense.